File: dev-docs/snippets/groups/delete.swift
import JsBaoClient

// Delete a group by its type and ID. Returns a typed `{ success: Bool }`.
func deleteGroup(client: JsBaoClient, groupType: String, groupId: String) async throws {
  // #region example
  let result = try await client.groups.delete(groupType: groupType, groupId: groupId)
  let success = result.success
  // #endregion example
  _ = success
}

File: dev-docs/snippets/groups/delete.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Delete a group by its type and ID. Returns `{ success: boolean }`.
export async function deleteGroup(
  client: JsBaoClient,
  groupType: string,
  groupId: string,
) {
  // #region example
  const { success } = await client.groups.delete(groupType, groupId);
  // #endregion example
  return success;
}
